### ISSUE TITLE
Meta quotes removed upon rebuilding rule

### ISSUE THREAD

**Issue by jgrunzweig** [plyara/plyara#22]
Noticed this today while playing this this library. Upon parsing and rebuilding a rule, the double quotes are removed in values within the meta section.

Example code:
`import plyara
ply = plyara.Plyara()

rule = """rule garbageRule
{
	meta:
		author = "Josh Grunzweig"

	strings:
		$garbage = "hi world"

	condition:
		any of them
}"""
p = ply.parse_string(rule)
print(repr(p))
print(ply.rebuild_yara_rule(p[0]))`

Output:
`[{'condition_terms': ['any', 'of', 'them'], 'raw_strings': 'strings:\n\t\t$garbage = "hi world"\n\n\t', 'raw_condition': 'condition:\n\t\tany of them\n', 'raw_meta': 'meta:\n\t\tauthor = "Josh Grunzweig"\n\n\t', 'rule_name': 'garbageRule', 'stop_line': 11, 'start_line': 1, 'strings': [{'name': '$garbage', 'value': '"hi world"'}], 'metadata': {'author': 'Josh Grunzweig'}}]
rule garbageRule {

	meta:
		author = Josh Grunzweig

	strings:
		$garbage = "hi world"

	condition:
		any of them
}

`

The unit test also appears to fail on this particular item:

======================================================================
FAIL: test_rebuild_yara_rule_metadata (__main__.TestStaticMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/unit_tests.py", line 82, in test_rebuild_yara_rule_metadata
    self.assertTrue('string_value = "TEST STRING"' in unparsed)
AssertionError: False is not true

----------------------------------------------------------------------

**Comment by jgrunzweig** [comment]
Pseudo false alarm. Functionality doesn't work on the pip install, but works correctly in master when building manually.

**Comment by rshipp** [comment]
This is odd, master should be the same as the latest version in pip, 1.2.6. If it hasn't been overwritten by your local install, could you grab the version you were using? And what Python version are you running?

    pip freeze | grep plyara
    python -V

I can't reproduce that test failure on the latest 2 versions under Python 2.7.

```
$ git checkout v1.2.5
Note: checking out 'v1.2.5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 5cf3725 Bump version to v1.2.5
$ nosetests
...................................
----------------------------------------------------------------------
Ran 35 tests in 1.274s

OK
$ git checkout v1.2.6
Note: checking out 'v1.2.6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at d231d38 Bump version to v1.2.6
$ nosetests
...................................
----------------------------------------------------------------------
Ran 35 tests in 1.264s

OK
```

**Comment by jgrunzweig** [comment]
Checking my logs, I think I understand what happened. I installed it a couple of months back and was running 1.2.3, however, I forgot I had done this. When I did a pip install earlier today it simply said I had the requirements met (I didn't realize until just now) and carried on. I just did a quick check and 1.2.6 is in fact installing via pip. This was simply stupidity on my part. 

So, tl;dr everything is good, but I made a mistake on my part.
